packages/phpstan-rules/packages/cognitive-complexity/tests/AstCognitiveComplexityAnalyzer/AstCognitiveComplexityAnalyzerTest.php

```diff
@@ -42,7 +42,7 @@ public function test(SmartFileInfo $fixtureFileInfo): void
     {
         $inputAndExpected = StaticFixtureSplitter::splitFileInfoToInputAndExpected($fixtureFileInfo);
 
-        $functionLike = $this->parseFileToFistFunctionLike($inputAndExpected->getInput());
+        $functionLike = $this->parseFileToFirstFunctionLike($inputAndExpected->getInput());
```

Comment:
fixed typo in method name: `Fist` -> `First`

Reply:
Thanks :+1: